src/components/CreateVmWizard/steps/SummaryReview.js

```diff
@@ -60,7 +62,17 @@ const ReviewBasic = ({ id, dataCenters, clusters, isos, templates, operatingSyst
     <Item id={`${id}-memory`} label={msg.memory()}>{userFormatOfBytes(basic.memory, 'MiB').str}</Item>
     <Item id={`${id}-cpus`} label={msg.cpus()}>
       { basic.cpus }
-      {/* TODO: Include topology (as a popover?) */}
+      <OverlayTooltip
+        id={`${id}-summary-vcpus-tooltip`}
+        tooltip={msg.totalCpuTooltip({
+          cores: basic.topology.cores,
+          sockets: basic.topology.sockets,
+          threads: basic.topology.threads,
+        })}
+        placement={'top'}
+      >
+        <InfoCircleIcon className={style['info-circle-icon']} />
```

Comment:
I used the `info-circle-icon` name with respect to the class from one of the Ben's PRs (I don't remember which one exactly) where he created a style which is the same as we need here. The name was chosen probably because the class is used with `InfoCircleIcon` component. So I don't think that changing it to `cpu-toplogy-tooltip-icon` would be more accurate.

Reply:
Humm...sounds like something to address in the tooltips PR #1289 for consistency sake.  In that case, you're ok with leaving it as-is.